common/libs/wallet/trezor.js

```diff
@@ -5,21 +5,25 @@ import Big from 'bignumber.js';
 import { addHexPrefix } from 'ethereumjs-util';
 import DeterministicWallet from './deterministic';
 import { stripHex } from 'libs/values';
-
 import type { RawTransaction } from 'libs/transaction';
 
+// Identical to ethFuncs.getNakedAddress
+function stripAndLower(str) {
```

Comment:
Any reason not to move into `libs/values`? Seems like this would be useful outside of just trezor.

Reply:
PrivKeyWallet actually implements it on the class:

```
export default class PrivKeyWallet implements IWallet {
...

  getNakedAddress(): Promise<string> {
    return new Promise(resolve => {
      this.getAddress().then(address => {
        resolve(stripHex(address));
      });
    });
  }
```

Reply:
We should probably figure out why that discrepancy exists (not that it needs to be fixed as a part of this PR)

Reply:
stripHex already lowercases the input 
```js
export function stripHex(address: string): string {
  return address.replace('0x', '').toLowerCase();
}
```

Reply:
If anything, the `stripHex` in `/libs/values.js` should be renamed to `stripHexAndLower` since thats what it actually does